ecosystem/sep-0039.md

```diff
@@ -0,0 +1,155 @@
+
+## Preamble
+
+```
+SEP: 0039
+Title: Interoperability Recommendations for NFTs
+Authors: Tomer Weller <tomer@stellar.org>, Frederic Rezeau <frederic.rezeau@gmail.com>
+Track: Informational
+Status: Active
```

Comment:
I think we should leave this in draft status for some period of time to let this develop a little? Or did we already discuss status somewhere and progressing it?

Reply:
I thought that was the point of this PR and all of the prior iterations around this topic :thinking: maybe FCP?

Reply:
Amended in https://github.com/stellar/stellar-protocol/pull/1140/commits/d8b3386b80bb12e110ee06b5d85b70edd6b148f2